lexer/lex_test.go

```diff
@@ -851,6 +884,10 @@ func prependOnTestCases(tcase TestCase, tcases []TestCase) []TestCase {
 	return newcases
 }
 
+func sfmt(format string, a ...interface{}) utf16.Str {
```

Comment:
lazy motherfocker =)

Reply:
auheuheuahuae man you don't forget anything =P. Learning with the master of all lazy motherfockers =P